### DISCUSSION TITLE
HW4 typo error

### DISCUSSION THREAD

**Discussion by Amir-M-Vahedi** [cooperrc/advanced-dynamics#37] [Logistics] [answered]
Hello everyone,

I think the requested formula of period for problem one is incorrect.
The requested form is (which is valid for the problem2 that has a pendulum arm):
$P=2\pi\sqrt{\frac{mg+2kr}{2kg}}$
But in the problem1 we have point mass and the formula will become:
 $P=2\pi\sqrt{\frac{mg+kr}{kg}}$

In the problem2, the mass in the question is 1kg but in the code it is written as 0.1, which one is correct?

**Comment by Amir-M-Vahedi** [comment]
I realized the first part of my question is wrong because we have to consider two spring forces.

Does anyone have any idea about the parameters of the problem 2?

**Reply by cooperrc** [reply, marked as answer]
This period is correct assuming each spring has stiffness $k$, 

$P=2\pi\sqrt{\frac{mg+2kr}{2kg}}$

If you consider the total stiffness of the x2 springs as say $k_{eff}$, then 
 $P=2\pi\sqrt{\frac{mg+k_{eff}r}{k_{eff}g}}$

> In the problem2, the mass in the question is 1kg but in the code it is written as 0.1, which one is correct? 

$m = 1~kg$ so you can change the code to match, 

`m = 1 # kg`

**Reply by Amir-M-Vahedi** [reply]
Thanks for your answer.
